joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 20

sting_gripper_controller:
    type: position_controllers/JointPositionController
    joint: sting_gripper_joint